MeetupPlatformApi/Migrations/20220329130316_AddMeetup.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+#nullable disable
+
+namespace MeetupPlatformApi.Migrations
+{
+    public partial class AddMeetup : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "Meetups",
+                columns: table => new
+                {
+                    Id = table.Column<int>(type: "integer", nullable: false)
+                        .Annotation("Npgsql:ValueGenerationStrategy", NpgsqlValueGenerationStrategy.IdentityByDefaultColumn),
+                    Name = table.Column<string>(type: "text", nullable: false),
+                    StartTime = table.Column<DateTime>(type: "timestamp with time zone", nullable: false),
```

Comment:
Don't just change migration like this (`nullable: false`). Instead You should update Entity (add `[Required]` annotation to `Name` and `Description` field) and then re-create this migration completely from scratch. If You don't do this, we have unexpected changes in upcoming migrations. To ensure, that all migration logic is encapsulated in this migration, You can create a temporary migration, check if it's empty and remove it (this way You can ensure that last changed didn't produce any phantom affect on DB schema)